<?xml version="1.0" encoding="utf-8"?>

<launch>
	<node name="agv_control" pkg="agv_control" type="agv_control" output="screen">
		<param name="Kp_Linear" type="double" value="3.0"/>
		<param name="Ki_Linear" type="double" value="0.0"/>
		<param name="Kd_Linear" type="double" value="0.5"/>
		<param name="Kp_Angular" type="double" value="5.0"/>
		<param name="Ki_Angular" type="double" value="0.0"/>
		<param name="Kd_Angular" type="double" value="1.0"/>
		<param name="toleranceLinear" type="double" value="0.01"/>
		<param name="toleranceAngular" type="double" value="0.005"/>
		<param name="linearMax" type="double" value="0.7"/>
		<param name="angularMax" type="double" value="1.0"/>
		<param name="publish_velocity_command_frequency" type="double" value="600.0"/>
		<param name="markerDistance" type="double" value="1.8"/>
		<param name="lookaheadDistance" type="double" value="0.5"/>
	</node>
</launch>
